src/lib/y2network/interface_config_builders/br.rb

```diff
@@ -0,0 +1,24 @@
+require "yast"
+require "y2network/interface_config_builder"
+
+Yast.import "NetworkInterfaces"
+
+module Y2Network
+  module InterfaceConfigBuilders
+    class Br < InterfaceConfigBuilder
+      def initialize
+        super(type: "br")
+      end
+
+      def bridgable_interfaces
+        Yast::LanItems.GetBridgeableInterfaces(name)
+      end
+
+      def already_configured?(devices)
+        devices.any? do |device|
+          ![nil, "none"].include?(Yast::NetworkInterfaces.devmap(device))
```

Comment:
this is wrong (my proposal was wrong) ... you need ```...devmap(device)[BOOTPROTO]``` bcs ```devmap``` returns the device's configuration (if any) and you need to check for BOOTPROTO option value.